src/LintDirtyModulesPlugin.js

```diff
@@ -2,6 +2,25 @@ import { isMatch } from 'micromatch';
 
 import linter from './linter';
 
+function mapFileTimestampsToOldFormat(fileTimestamps) {
+  for (const [filename, FileSystemInfoEntry] of fileTimestamps.entries()) {
+    fileTimestamps.set(
+      filename,
+      FileSystemInfoEntry ? FileSystemInfoEntry.timestamp : null
+    );
+  }
+
+  return fileTimestamps;
+}
```

Comment:
We have a big loss here mapping all the entries.
We can just do a simple check on `hasFileChanged`
No need this function

Reply:
yep, you are right